enterprise/backend/test/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions_test.clj

```diff
@@ -1045,7 +1048,7 @@
                        :aggregation [[:sum $total]]
                        :breakout    [&P.people.source
                                      $product_id->products.category]
-                       :limit       5}))))))))))
+                       :limit       10}))))))))))
```

Comment:
The query is actually supposed to return 6 rows total (split across 4 subqueries I think) so having an overall limit of 5 means you won't see all the rows. The old impl applied the limit separately to each subquery. Now that we have just one query I had to fix this test